storybook/src/components/IconButton/IconButton.docs.mdx

```diff
@@ -14,22 +14,6 @@ import README from "../../../../packages/css/src/components/icon-button/README.m
 
 ## Examples
 
-### Level 3
```

Comment:
Don't we want to show the different sizes here? Maybe similar to how we do in the Icon docs

Reply:
We decided against this as Icon Button is on its way to deprecation anyway.